portal-impl/src/com/liferay/portal/dao/orm/hibernate/event/MVCCSynchronizerPostUpdateEventListener.java

```diff
@@ -33,48 +31,23 @@ public void onPostUpdate(PostUpdateEvent postUpdateEvent) {
 		Object entity = postUpdateEvent.getEntity();
 
 		if (entity instanceof MVCCModel) {
-			MVCCModel mvccModel = (MVCCModel)entity;
-
-			long mvccVersion = mvccModel.getMvccVersion();
-
-			Class<?> modelClass = entity.getClass();
-
-			BaseModel<?> baseModel = (BaseModel<?>)entity;
-
-			Serializable primaryKeyObj = baseModel.getPrimaryKeyObj();
-
 			if (entity instanceof CTModel) {
 				CTModel<?> ctModel = (CTModel<?>)entity;
 
 				if (ctModel.getCtCollectionId() != 0) {
```

Comment:
I think we need to remove this check completely or else we will end up with stale object exceptions

Reply:
Yes, you are right, we should. I missed this part